test-tests/shared/src/test/scala/zio/test/BoolAlgebraSpec.scala

```diff
@@ -1,34 +1,30 @@
 package zio.test
 
+import zio.random.Random
 import zio.test.Assertion._
 import zio.test.BoolAlgebraSpecHelper._
-import zio.test.TestAspect._
-
-import scala.util.Random
 
 object BoolAlgebraSpec
     extends ZIOBaseSpec(
       suite("BoolAlgebraSpec")(
         test("all returns conjunction of values") {
           assert(BoolAlgebra.all(List(success1, failure1, failure2)), isSome(isFailure))
         },
-        test("and distributes over or") {
-          val a = randomBoolAlgebra()
-          val b = randomBoolAlgebra()
-          val c = randomBoolAlgebra()
-          assert(a && (b || c), equalTo((a && b) || (a && c)))
-        } @@ nonFlaky(100),
-        test("and is associative") {
-          val a = randomBoolAlgebra()
-          val b = randomBoolAlgebra()
-          val c = randomBoolAlgebra()
-          assert((a && b) && c, equalTo(a && (b && c)))
-        } @@ nonFlaky(100),
-        test("and is commutative") {
-          val a = randomBoolAlgebra()
-          val b = randomBoolAlgebra()
-          assert(a && b, equalTo(b && a))
-        } @@ nonFlaky(100),
+        testM("and distributes over or") {
+          zio.test.check(boolAlgebra, boolAlgebra, boolAlgebra) { (a, b, c) =>
```

Comment:
getting some weird compilation error without fully-qualified name

Reply:
Yes, it is because we have the `test` package object extend the `CheckVariants` trait so we don't clutter up the package object with all the different variants of the check methods. It looks like this issue only appears because the test suite uses the same package name. Not great but I don't have a better solution right now so we will have to go with it.